* grub-core/kern/emu/hostdisk.c (grub_util_biosdisk_write): Fix opening
  mode.

2012-04-18  Vladimir Serbinenko  <address@hidden>

--- grub-core/kern/emu/hostdisk.c       2012-04-18 21:48:52 +0000
+++ grub-core/kern/emu/hostdisk.c       2012-04-19 18:35:06 +0000
@@ -1081,7 +1081,7 @@
     {
       int fd;
       grub_disk_addr_t max = ~0ULL;
-      fd = open_device (disk, sector, O_RDONLY, &max);
+      fd = open_device (disk, sector, O_WRONLY, &max);
       if (fd < 0)
        return grub_errno;
